Makefile

```diff
@@ -1,54 +1,16 @@
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
+.PHONY: build clean test
 
-SETUP = ocaml setup.ml
+build:
+	jbuilder build @install
```

Comment:
its not --dev clean yet, due to various warnings. will add once its clean on all supported versions